### PR TITLE
Handful bricks

### PR DESCRIPTION
**Pull request by joatom**
Initial version of "A handful bricks - from SQL to Pandas."

Hello Abhishek,
I hope this article is not to basic / nonsense. 
Regards, Johannes

### CODE REVIEW

File: users/joatom/a-handful-of-bricks-from-sql-to-pandas/.ipynb_checkpoints/ARTICLE-checkpoint.md [hunk 1]
```diff
@@ -0,0 +1,2227 @@
+# Table of contents
```
Comment (abhishekkrthakur) [inline users/joatom/a-handful-of-bricks-from-sql-to-pandas/.ipynb_checkpoints/ARTICLE-checkpoint.md:1]: remove this file

File: users/joatom/a-handful-of-bricks-from-sql-to-pandas/ARTICLE.md [hunk 2]
```diff
@@ -0,0 +1,2227 @@
+# Table of contents
```
Comment (abhishekkrthakur) [inline users/joatom/a-handful-of-bricks-from-sql-to-pandas/ARTICLE.md:1]: although, we do have title file, you still need to add title here and please convert other titles to subtitles by using "##"

File: users/joatom/a-handful-of-bricks-from-sql-to-pandas/assets/Readme.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# References
```
Comment (abhishekkrthakur) [inline users/joatom/a-handful-of-bricks-from-sql-to-pandas/assets/Readme.md:1]: We dont need it. Maybe add it at the end of the article and remove this file

File: users/joatom/gihub_username [hunk 4]
```diff
@@ -0,0 +1 @@
+joatom
```
Comment (abhishekkrthakur) [inline users/joatom/gihub_username:1]: dont need this file. :) please remove it

**Review by abhishekkrthakur — CHANGES_REQUESTED** [review]

**Comment by abhishekkrthakur** [comment]
> I hope this article is not to basic / nonsense.
haha...its pretty good! Added a few comments that need to be fixed before merging :)

**Comment by joatom** [comment]
do you mind to add `.ipynb_checkpoints` to the central .gitignore? or am I allowed to put my customized .gitignore to my users folder?

**Comment by abhishekkrthakur** [comment]
Feel free to add it to central .gitignore :)

**Comment by abhishekkrthakur** [comment]
merged. ill add it to central .gitignore

**Pull request merged**